system/Commands/Utilities/Routes/AutoRouterImproved/ControllerMethodReader.php

```diff
@@ -113,6 +117,12 @@ public function read(string $class, string $defaultController = 'Home', string $
                         $params[$param->getName()] = $required;
                     }
 
+                    // If it is the default controller, the method will not be
+                    // routed.
+                    if ($classShortname === $defaultController) {
+                        $route = 'x ' . $route;
```

Comment:
is `x` is some special name?

Reply:
Yes. I want to show the route is invalid and will not be routed. Is there anything good?

Reply:
sounds good 👍